### ISSUE TITLE
Filter records with same position

### ISSUE THREAD

**Issue by gdrako** [tananaev/traccar#2948]
Hello

It is possible to apply in app server a new filter that cuts to the entries that have the same lon and lat?
This would be very useful because such records I think in most cases it is useless.

Thank you

**Comment by Abyss777** [comment]
Hello
You can use `filter.distance` with some small value https://www.traccar.org/configuration-file/

**Comment by gdrako** [comment]
Look ..All the parameters present in the configuration have to do with the distance and time between records.
I mean a filter to control the previous position and the current though the lat, lon is the same to cut it.
This filter is not influenced by other parameters such as distance or limit time.

**Comment by tananaev** [comment]
If the GPS module has satellite signal, the position is almost never exactly the same. If there is no signal, the fix time should be the same and you can use duplicate filtering.

**Comment by gdrako** [comment]
The problem I have is that several devices have the same position in too many records.
This occurs when the devices are static.
I could use the filter.static when the speed is equal to 0 to the filter them.
If this filter cut all entries with 0 speed except the first would be for me the best option and would solve my problem.
I say this because the first recording with speed =0 I need it for the parking reports
Now I see in the code cuts all records from any record ..
private boolean filterStatic(Position position) {
        return filterStatic && position.getSpeed() == 0.0;
}
How I can do to cut each position.getDeviceId entries except the first ?

**Comment by tananaev** [comment]
Is `fixTime` different for those positions?

**Comment by gdrako** [comment]
Yes

**Comment by tananaev** [comment]
Then I think @Abyss777's suggestion is your best option.

**Comment by gdrako** [comment]
Yes filter.distance do that but only if poisitions is within the limit time.
If positions are outside of the limit time then the records will pass on database

**Comment by tananaev** [comment]
What time limit are you talking about?

**Comment by gdrako** [comment]
I mean the time we put the following parameter.
<entry key='filter.limit'>130</entry>

**Comment by tananaev** [comment]
Well, just don't put that parameter.
